osg_configure/configure_modules/misc.py

```diff
@@ -108,8 +108,10 @@ def configure(self, attributes):
 
         # run fetch-crl script
         if not utilities.fetch_crl():
-            self.log("Error while running fetch-crl script", level=logging.ERROR)
-            raise exceptions.ConfigureError('fetch-crl returned non-zero exit code')
+            self.log("Error while running fetch-crl script", level=logging.WARNING)
+
+        if not utilities.crls_exist():
+            self.log("No CRLs exist and Fetch-CRL couldn't download any.  GSI authentication is likely to fail.  To attempt another download, run 'fetch-crl'.", level=logging.WARNING)
```

Comment:
Is this message true? It's possible that there was an error in fetching just one CRL, which could potentially be fine

Reply:
`if not utilities.crls_exist()...`
That message is only printed if _no_ CRLs exist, i.e. there are no *.r0 files.